src/scrapers/hapoalim.js

```diff
@@ -24,6 +24,15 @@ function convertTransactions(txns) {
   });
 }
 
+function fetchSummary(balanceCreditResult) {
+  return {
+    balance: parseFloat(parseFloat(balanceCreditResult.currentBalance).toFixed(2)),
+    creditLimit: parseFloat(parseFloat(balanceCreditResult.currentAccountLimitsAmount).toFixed(2)),
+    creditUtilization: parseFloat(parseFloat(balanceCreditResult.withdrawalBalance).toFixed(2)),
```

Comment:
@chilikla are you sure `balanceCreditResult.withdrawalBalance` is the right prop to use for `creditUtilization`?
I don't have an Hapoalim account, so I have no way of seeing what it represents.

Reply:
@eshaham it's a good question. This property is saying how much left for you to withdraw. So if you are in overdraft, this is `creditUtilization` per se. But if you have plus in your bank account, it's  a sum of your `balance` and `creditLimit`.
What do you think will be preferred in case of plus - returning 0 (since the utilization is zero right now) or the current sum?

Reply:
@chilikla in that case, in case of an overdraft, shouldn't `creditUtilization` be `-balance`? I know we said something different in #76, but it seems to me this is the right thing in case of a bank, isn't it?
In case of a positive balance, I think it should be 0.

Reply:
@eshaham there is `balanceCreditResult.creditLimitUtilizationAmount`, which indeed has 0 for me right now (since I'm not in overdraft). I think it's logical to use it. I just don't know what will its value in case of overdraft. What do you think?

Reply:
@chilikla hmmm, sounds like it has the right name...
We can either try to use it, or do `-balance`.
Your call 😄

Reply:
@eshaham I think `creditLimitUtilizationAmount` is better, since I'm not sure `-balance` is the right value for utilization but how much left to withdraw from the account, or `creditLimit + balance` (in case of negative value for `balance`).

Reply:
@chilikla credit utilization should be how much you've used off your credit limit, so it should probably be something like `Math.max(-balance, creditLimit)`. I wonder what happens in the bank when you go over your credit limit... Anyhow, I guess `creditLimitUtilizationAmount` has the right name, so we should probably go with that... 😄